server/endpoints_v1.go

```diff
@@ -163,18 +163,22 @@ func (server *HTTPServer) addV1RuleEndpointsToRouter(router *mux.Router, apiPref
 
 	router.HandleFunc(apiPrefix+DisableRuleForClusterEndpoint, server.proxyTo(
 		aggregatorBaseEndpoint,
-		nil,
+		&ProxyOptions{RequestModifiers: []RequestModifier{
+			server.newExtractUserIDFromTokenToURLRequestModifier(ira_server.DisableRuleForClusterEndpoint),
+		}},
 	)).Methods(http.MethodPut, http.MethodOptions)
 
 	router.HandleFunc(apiPrefix+EnableRuleForClusterEndpoint, server.proxyTo(
 		aggregatorBaseEndpoint,
-		nil,
+		&ProxyOptions{RequestModifiers: []RequestModifier{
+			server.newExtractUserIDFromTokenToURLRequestModifier(ira_server.EnableRuleForClusterEndpoint),
+		}},
 	)).Methods(http.MethodPut, http.MethodOptions)
 
 	router.HandleFunc(apiPrefix+DisableRuleFeedbackEndpoint, server.proxyTo(
 		aggregatorBaseEndpoint,
 		&ProxyOptions{RequestModifiers: []RequestModifier{
-			server.newExtractUserIDFromTokenToURLRequestModifier(ira_server.DisableRuleFeedbackEndpoint),
```

Comment:
I'm probably missing something, but shouldn't this stay the same? This is the "disable feedback" endpoint `DisableRuleFeedbackEndpoint`

Reply:
Oh, I misunderstood your comment, sorry.

You are right, running to fix it